h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -374,5 +530,60 @@ describe('SearchBarController', function () {
           done();
         });
     });
+
+    describe('resolving initial group lozenge and input value', function(){
```

Comment:
I found this difficult to parse. Do you mean "converting server-rendered group lozenges to client-rendered group lozenges"?